components/HighlightedLinks/src/index.scss

```diff
@@ -0,0 +1,41 @@
+.denhaag-highlighted-links {
+  background-color: var(--denhaag-highlighted-links-background-color);
+  padding-block-start: var(--denhaag-highlighted-links-padding-block-start);
+  padding-block-end: var(--denhaag-highlighted-links-padding-block-end);
+  width: var(--denhaag-highlighted-links-width);
+}
+
+.denhaag-highlighted-links__container {
```

Comment:
This block will be edited from the editor, therefor it's already within a container. I believe this container is redundant. Please check with design.